core/tests/protractor_desktop/explorationHistoryTab.js

```diff
@@ -601,6 +603,62 @@ describe('Exploration history', function() {
     await users.logout();
   });
 
+  it('should show the history of exploration metadata', async function() {
```

Comment:
Can you please provide the links for 5 straight successfull runs of the suite that contains this test?

Reply:
I am running them now. Might take some time as all runners are busy right now. Till now, I have two straight successful runs (will keep updating them as the checks run):

[Attempt 1](https://github.com/oppia/oppia/actions/runs/2601513613/attempts/1)
[Attempt 2](https://github.com/oppia/oppia/actions/runs/2601513613/attempts/2)

Reply:
Please attach it in the PR description too.

Reply:
Done.